salt/pillar/file_tree.py

```diff
@@ -189,6 +196,9 @@
 # Set up logging
 log = logging.getLogger(__name__)
 
+# The default option values
+__opts__ = {}
```

Comment:
Why are you overwriting the packed opts dunder dictionary?

Reply:
I originally borrowed this from hg_pillar due to an error I encountered when writing the test case. Now I'm not getting the error in my test case, so it doesn't seem to be necessary any more. I've therefore removed it from the latest commit.

Note that this was only initializing the file_tree.__opts__ dictionary, not overwriting the global one.

Reply:
Yeah, I understood that. It shouldn't have been done in hg_pillar either, likely.